content/kotlin/concepts/math-methods/terms/acos/acos.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '.acos()'
+Description: 'Calculates the arccosine or reverse cosine of a value, returning the angle in radians.'
```

Comment:
should i also include inverse cosine?

Description: 'Calculates the arccosine or reverse cosine or inverse cosine of a value, returning the angle in radians.'